chunker/scripts/runBenchmarkTest.sh

```diff
@@ -17,7 +17,7 @@ for JAR in `ls $LIB`; do
 done
 
 
-TESTDATA="src/main/resources/test.txt"
+TESTDATA="/shared/corpora/corporaWeb/written/eng/chunking/conll2000distributions/test.txt"
```

Comment:
Yes. But runBenchmarkTest.sh is not used anyhow, you can either adopt it or reject it.